lib/parsers/package-lock-parser.ts

```diff
@@ -120,8 +120,13 @@ export class PackageLockParser implements LockfileParser {
 
     // number of dependencies including root one
     let treeSize = 1;
-
+    const treeSizeLimit = 6.0e6;
+  
     for (const dep of topLevelDeps) {
+      // On Call we decided that the tree size limit will be of 6 millions.
```

Comment:
This might be better next to the definition of the const
```suggestion
      // oncall decided tree size limit should be 6 million
```